### PR TITLE
New package: MinnowIT.Foldr version 3.1.0

### PR DESCRIPTION
**Pull request by mibmit**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/171143)

### CODE REVIEW

File: manifests/m/MinnowIT/Foldr/3.1.0/MinnowIT.Foldr.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: MinnowIT.Foldr
+PackageVersion: 3.1.0
+PackageLocale: en-US
+Publisher: MinnowIT
+PackageName: Foldr
+License: Licence
+Copyright: © Microsoft Corporation. All rights reserved.
```
Comment (a-mnich) [inline manifests/m/MinnowIT/Foldr/3.1.0/MinnowIT.Foldr.locale.en-US.yaml:10]:
```suggestion
Copyright: © Minnow IT LTD
```

**Review by a-mnich — CHANGES_REQUESTED** [review]
The publisher has the copyright.

File: manifests/m/MinnowIT/Foldr/3.1.0/MinnowIT.Foldr.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: MinnowIT.Foldr
+PackageVersion: 3.1.0
+PackageLocale: en-US
+Publisher: MinnowIT
+PackageName: Foldr
+License: Licence
```
Comment (a-mnich) [inline manifests/m/MinnowIT/Foldr/3.1.0/MinnowIT.Foldr.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by a-mnich — CHANGES_REQUESTED** [review]
see change regarding license

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 894.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 895.)

**Pull request closed without merging**